package/src/components/MessageList/MessageSystem.tsx

```diff
@@ -7,14 +7,21 @@ import { useTheme } from '../../contexts/themeContext/ThemeContext';
 import { useTranslationContext } from '../../contexts/translationContext/TranslationContext';
 
 import type { DefaultStreamChatGenerics } from '../../types/types';
-import { getDateString } from '../../utils/getDateString';
+import { getDateString } from '../../utils/i18n/getDateString';
 
 export type MessageSystemProps<
   StreamChatGenerics extends DefaultStreamChatGenerics = DefaultStreamChatGenerics,
 > = {
   /** Current [message object](https://getstream.io/chat/docs/#message_format) */
   message: MessageType<StreamChatGenerics>;
+  /**
+   * Additional styles for the system message container.
+   */
   style?: StyleProp<ViewStyle>;
+  /*
+   * Lookup key in the language corresponding translations sheet to perform date formatting
+   */
+  timestampTranslationKey?: string;
```

Comment:
do we really need this prop to be customisable.. cant we just always use the same 'timestamp/MessageSystem' key here ?

Reply:
It's up to integrators to customize this prop if they want a shorter name. React SDK has added this support, so I added it as well. We can choose not to give this flexibility though.

Reply:
Thinking about this we may not need so high degree of customization. I would remove it.

Reply:
I have removed this prop from components in React SDK

Reply:
Removed on RN SDK as well